src/App/Expression.cpp

```diff
@@ -2463,7 +2462,7 @@ Expression *FunctionExpression::_copy() const
 
 void FunctionExpression::_visit(ExpressionVisitor &v)
 {
-    std::vector<Expression*>::const_iterator i = args.begin();
+    auto i = args.begin();
```

Comment:
Lost `const`-ness

Reply:
But looking at the entire function:
```
void FunctionExpression::_visit(ExpressionVisitor &v)
{
    auto i = args.begin();

    HiddenReference ref(f == HIDDENREF || f == HREF);
    while (i != args.end()) {
        (*i)->visit(v);
        ++i;
    }
}
```
So i can't be const. No?

Reply:
`i` can't be `const`, but its type can be a `const_iterator`. Try using `std::as_const`, something like (untested):
```
auto i = std::as_const(args).begin();
```

Reply:
What if use `args.cbegin()` to make a const_iterator?

Reply:
Yep.